### ISSUE TITLE
Animation does not work and I can't figure out what is wrong.

### ISSUE THREAD

**Issue by natalie-macpherson** [natalie-macpherson/dsn1676-term-project#1]
I don't know what my chair is not rocking back and forth. Am I using the wrong element? Do you see any mistakes because it does not work in both Firefox or Chrome. In the dev. tool it says its an unknown property.

Side note**
Also, in class when just work with one screen size, so how does absolute work with responsiveness? My object (the chair) will always be in the same place. Is there where we would use media queries?

https://github.com/natalie-macpherson/dsn1676-term-project/blob/gh-pages/css/main.css#L70

@thomasjbradley

**Comment by thomasjbradley** [comment]
I can see two things:
1. `animation` is spelled wrong on the line you highlighted, and the line below
2. Inside `@-webkit-keyframes`, the `transform` properties should have the Webkit prefix, like: `-webkit-transform`

**Comment by natalie-macpherson** [comment]
Thank you Thomas for pointing out those mistakes. I made the changes but the chair still wont rock. I have continued to play around with it but the movement will still not happen.

@thomasjbradley

**Comment by thomasjbradley** [comment]
I found another one for you: `@keyframes` is missing the final "s".

**Comment by natalie-macpherson** [comment]
yep I saw that one and change it hoping it would solve the issue. I just didn't commit the change yet. It has been committed now but it's still not working. I have ask some people in the class if they can find anything but still no luck. Am I using the correct elements in order to make the chair rock in the first place?

On a positive note my second object moves so at least I know that I am doing something right.

@thomasjbradley

**Comment by natalie-macpherson** [comment]
Nevermind, I figured it out.
https://github.com/natalie-macpherson/dsn1676-term-project/blob/gh-pages/css/main.css#L76

What I did have was:

0% {-webkit-transform: rock(-45deg);} etc...

but I now changed it and it works.

Thank you Thomas for pointing out the typos. I find that I am making more of them since I have switched over to Sublime. I wish there was a way for it to spell check.

@thomasjbradley
